api/suggestion/models/suggestion.js

```diff
@@ -1,24 +1,48 @@
 "use strict";
 const { createLifecycleHooks } = require("../../../utils/algolia");
 
-/**
- * Read the documentation (https://strapi.io/documentation/developer-docs/latest/development/backend-customization.html#lifecycle-hooks)
- * to customize this model
- */
 const contentType = "suggestion";
 
+const notifyAboutSuggestion = async (suggestion) => {
+  const settings = await getSettings();
+
+  const recipients = settings.suggestion_notification_recipients?.split(",");
+
+  await Promise.all(
+    recipients.map(
+      async (recipient) =>
+        await sendSuggestionNotification(recipient, suggestion)
+    )
+  );
+};
+
+const getSettings = async () => {
+  return await strapi.services.settings.find();
+};
+
+const sendSuggestionNotification = async (recipient, suggestion) => {
+  console.log("Sending new suggestion notification to:", recipient);
+
+  const emailConfig = {
+    to: recipient,
+    subject: "Uusi toteutusehdotus",
+    text: `Partio-ohjelmaan on lähetetty uusi toteutusehdotus: ${suggestion.title}, kirjoittaja: ${suggestion.author}.`,
+  };
+
+  await strapi.plugins.email.services.email.send(emailConfig);
+};
+
```

Comment:
Do we have other cases where we want to send emails? Just wondering whether it would make sense to make a util absraction for sending email ie. `sendBatchEmail(recipients, settings)` and `sendEmail(recipient, settings)`. If this is the only case of use sending emails this way I thinks its fine to leave it here as is.

Reply:
Currently I think this is the only place where we need to send emails ourselves. I say let's leave it as is for now, and make an abstraction later if needed.